spec/common-client-libs-sections.json

```diff
@@ -13,12 +13,19 @@
     "type": "markdown",
     "libs": ["js", "js_v1", "dart", "dart_v0"]
   },
+  {
+    "title": "Initializing",
+    "id": "initializing",
+    "slug": "initializing",
+    "type": "markdown",
+    "libs": ["dart", "dart_v0"]
+  },
   {
     "title": "Typescript Support",
     "id": "typescript-support",
     "slug": "typescript-support",
     "type": "markdown",
-    "libs": ["js", "js_v1", "dart", "dart_v0"]
+    "libs": ["js", "js_v1"]
```

Comment:
Hmm, `Typescript Support` still appears in the side bar. Any idea how I could remove it from Dart docs?
<img width="557" alt="Screen Shot 2022-12-13 at 2 54 58" src="https://user-images.githubusercontent.com/18113850/207118727-47f81130-320a-4eab-bb4f-ae3308927648.png">

Reply:
ahh yea, ill mention it to Terry. it's a bug

Reply:
Cool!

I will add some other Flutter docs specific fixes mentioned in slack to this PR while this bug is being worked on!

Reply:
we'll merge anyway. separate issue

Reply:
Sounds good! I'm good to merge this one right now!

Reply:
@MildTomato